index.html

```diff
@@ -22,25 +23,195 @@
+                        <li class="javascript">javaScript</li>
+                    </ul>
+                    <button class="btn" type="button">See Project</button>
+                </div>
+            </div>
+
+
+        </section>
+        <!-- /*About myself ----------------------------section*/ -->
+        <section class="myself">
+            <div class="Top-part">
+                <h3>About Myself</h3>
+                <p class="self">
+                    Hello I'm a software developer! I can help
+                    you build a product, feature or website.
+                    Look through some of my work and
+                    experience! if you like what you see and
+                    have a project you need coded, don't
+                    hesitate to contact me.
+                </p>
+                <p class="connect">LET'S CONNECT!</p>
+                <ul class="social-media">
+                    <li><a href="#"><img src="./Images/Github.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./Images/Linkedin.svg" alt="Linkedin icon"></a></li>
+                    <li><a href="#"><img src="./Images/peace.svg" alt="peace icon"></a></li>
+                    <li><a href="#"><img src="./Images/twitter.svg" alt="twitter icon"></a></li>
+                    <li><a href="#"><img src="./Images/Mirror.svg" alt="Mirror icon"></a></li>
+                </ul>
+                <button type="button">Get my resume</button>
+            </div>
+
+            <div class="still-self">
+                <ul class="stacks">
+                    <li>
+                        <div class="lang-title">
+                            <h3 class="Frameworks">Languages</h3>
+                            <img src="./images/dropdown.png" alt="">
+                        </div>
+                        <ul class="small-ul">
+                            <li>
+                                <img src="./images/javascript.png" alt="">
+                                <h3 class="lang-names">javascript</h3>
+                            </li>
+                            <li>
+                                <img src="./images/html.png" alt="">
+                                <h3 class="lang-names">HTML</h3>
+                            </li>
+                            <li>
+                                <img src="./images/css.png" alt="">
+                                <h3 class="lang-names">css</h3>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="frames">
+                        <h3 class="Frameworks">Frameworks</h3>
+                        <img class="vector" src="./images/icons8-chevron-right-26.png" alt="">
+                    </li>
+                    <li class="frames">
+                    <h3 class="Frameworks">Skills</h3>
+                    <img class="vector" src="./images/icons8-chevron-right-26.png" alt="">
```

Comment:
- [ ] While you did a good job creating all the elements there is an error with your images because they are not been calling. Maybe one reason could be that your all calling each image (inside src) with `./images` instead of using the capitalized file `./Images` :+1: , please try to fix that :pray: 

**Your design**'

![Screenshot from 2023-01-14 08-13-58](https://user-images.githubusercontent.com/105117832/212473511-783d55d7-a741-4c39-a7d0-67f18ca79a3b.png)

**Figma design**

![Screenshot from 2023-01-14 08-18-10](https://user-images.githubusercontent.com/105117832/212473557-18622e51-ed0e-42e8-823d-d5bc1d83a26e.png)